pkg/daemon/image-inspect.go

```diff
@@ -0,0 +1,83 @@
+package daemon
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strings"
+	"time"
+
+	"github.com/containers/image/v5/docker"
+	"github.com/containers/image/v5/image"
+	"github.com/containers/image/v5/types"
+	"github.com/pkg/errors"
+)
+
+func retryIfNecessary(ctx context.Context, operation func() error) error {
```

Comment:
Any reason not to use [k8s.io/client-go/util/retry](https://pkg.go.dev/k8s.io/client-go/util/retry)?

Reply:
I followed the convention of what skopeo has been doing to retry https://github.com/containers/skopeo/blob/153f18dc0ac34e1adfc700806ef30af33187908b/cmd/skopeo/utils.go#L330